Modules/Generators/PackageGenerator/Sources/PackageGenerator/PackageGenerator.swift

```diff
@@ -72,7 +72,14 @@ public struct PackageGenerator: PackageGeneratorProtocol {
         let path = try createFolderIfNecessary(folder: "Sources", at: url, withName: name)
         try createMacroSourceFile(name: name, atPath: path)
     }
+    
+    public func runExternalScriptIfNeeded(at url: URL) throws {
+        guard fileManager.fileExists(atPath: url.path) else { return }
 
+        let shell = Shell(verbose: true)
+        _ = try shell.executeScript(at: url.absoluteURL.absoluteString)
```

Comment:
The `executeScript` function already is annotated with `@discardableResult`, so you can remove the `_ = `.